ospd_openvas/nvticache.py

```diff
@@ -20,14 +20,22 @@
 """ Provide functions to handle NVT Info Cache. """
 
 import logging
+import subprocess
+import sys
+
+from distutils.version import StrictVersion
 
 from ospd_openvas.db import NVT_META_FIELDS
+from ospd_openvas.errors import OspdOpenvasError
+
 
 logger = logging.getLogger(__name__)
 
 LIST_FIRST_POS = 0
 LIST_LAST_POS = -1
 
+SUPPORTED_NVTICACHE_VERSION = ['20.4']
```

Comment:
It should be named SUPPORTED_NVTICACHE_VERSION**S** and be a tuple to avoid changing the list. Tuples are immutable.

```suggestion
SUPPORTED_NVTICACHE_VERSIONS = ('20.4',)
```